packages/wdio-cli/tests/utils.test.ts

```diff
@@ -385,19 +385,59 @@ test('validateServiceAnswers', () => {
 })
 
 describe('getCapabilities', () => {
+    afterEach(() => {
+        jest.mock('@wdio/config')
```

Comment:
No need to mock, Jest automatically looks into all `__mocks__` directories and it therefor will find the mock here: `/packages/wdio-config/test/__mocks__/@wdio/config.ts`.
```suggestion
```